### ISSUE TITLE
Feature Request: Move / copy posts

### ISSUE THREAD

**Issue by mittalyashu** [netlify/netlify-cms#1575]
**Do you want to request a *feature* or report a *bug*?** feature

**What is the current behavior?**  Only edit and publish the post

**What is the expected behavior?** Move or copy the post to different directory

**Please mention your versions where applicable.**
* Netlify CMS version: 2.0.8
* Browser version: Chrome 68.0.3440.84

**Comment by erquhart** [comment]
Can you explain further what the request is here?

**Comment by mittalyashu** [comment]
Longer way _(using editor or explorer)_:

1. Open the editor or explorer
2. Move/Copy the file
3. Open the editor or terminal
4. Commit and push the changes to git
5. The website starts building
6. Changes are live

Shorter way _(using NetlifyCMS)_:

1. Open netlifycms
2. Click on the action _(move/copy)_ on the the post
3. Changes are live

**Comment by erquhart** [comment]
Why do you need to move content files manually? I need a lot more information here :) Context? What are you trying to accomplish?

**Comment by mittalyashu** [comment]
**NOTE:** Move and copy feature is related to repository only, it just to easily organize our posts from directory to directory, website visitors might see no change.

> Since we are using Jekyll for our [blogging website](https://blog.codecarrot.net), so I am saying this in the reference of Jekyll.

## Front-end

Here's the jekyll configuration `_config.yml`:

```yaml
permalink: /:title/
paginate_path: "/page/:num/"
```

When the website is build the posts are accessible via: `https://blog.codecarrot.net/{post.title}`, according to the `_config.yml` file `permalink`.

**For example:**

1. https://blog.codecarrot.net/how-should-i-name-my-git-commit-messages/
2. https://blog.codecarrot.net/working-with-files-and-folders/
3. https://blog.codecarrot.net/the-best-chrome-extensions-for-web-professionals/

## Behind the scene

Here' the `gif` of the tree structure from our blogging website _(link to repo: https://github.com/codecarrotnet/blog.codecarrot.net)_, as you can see in the `gif` our tree structire is quite deep in the `_post` directory.

![2018-08-25_18-31-14](https://user-images.githubusercontent.com/29014463/44618529-2b02d700-a895-11e8-8520-b17b712baf3a.gif)

> Currently we are not using NetlifyCMS in our blogging website, since NetlifyCMS doesn't support reading files from sub-directory.

We are a heavy user of `_draft` directory and once the post is ready to publish we move it to the desired category folder.

We are organized the posts based on categories. Currently we have more than 350 posts and organizing the posts from editor or explorer can be difficult sometimes.

**Comment by erquhart** [comment]
I definitely see the challenge, but it feels out of scope for Netlify CMS. E.g. it feels like you'd still have to solve this problem if you weren't using Netlify CMS.

**Comment by mittalyashu** [comment]
> it feels like you'd still have to solve this problem if you weren't using Netlify CMS.

Yeah 😅

> it feels out of scope for Netlify CMS

How it is out of scope for NetlifyCMS? If we can delete a post, using NetlifyCMS. Why not copy, move, or duplicate it?

**Comment by erquhart** [comment]
Copying and duplicating are on the roadmap. Moving seems to be the core request here, that's the part that's out of scope. Netlify CMS isn't a general file tree manager.

Closing this, but you can track the move/copy issue here: https://github.com/netlify/netlify-cms/issues/981.

**Comment by erquhart** [comment]
One more thing to add, though: eventually I'd like to see us allow extensions to create entire interfaces within Netlify CMS, in which case you _could_ build a file tree manager if you wanted that functionality. This isn't likely to happen anytime soon, but you can see my thoughts on it here: https://github.com/netlify/netlify-cms/issues/1193#issuecomment-376692788.
